src/data-model-providers/codegen/tests/TestCodegenModelViaMocks.cpp

```diff
@@ -542,9 +542,13 @@ const MockNodeConfig gTestNodeConfig({
             MOCK_ATTRIBUTE_CONFIG_NULLABLE(ZCL_ELAPSED_S_ATTRIBUTE_TYPE),
             MOCK_ATTRIBUTE_CONFIG_NULLABLE(ZCL_TEMPERATURE_ATTRIBUTE_TYPE),
             MOCK_ATTRIBUTE_CONFIG_NULLABLE(ZCL_POWER_MW_ATTRIBUTE_TYPE),
+            MOCK_ATTRIBUTE_CONFIG_NULLABLE(ZCL_POWER_MVA_ATTRIBUTE_TYPE),
+            MOCK_ATTRIBUTE_CONFIG_NULLABLE(ZCL_POWER_MVAR_ATTRIBUTE_TYPE),
             MOCK_ATTRIBUTE_CONFIG_NULLABLE(ZCL_AMPERAGE_MA_ATTRIBUTE_TYPE),
             MOCK_ATTRIBUTE_CONFIG_NULLABLE(ZCL_VOLTAGE_MV_ATTRIBUTE_TYPE),
             MOCK_ATTRIBUTE_CONFIG_NULLABLE(ZCL_ENERGY_MWH_ATTRIBUTE_TYPE),
+            MOCK_ATTRIBUTE_CONFIG_NULLABLE(ZCL_ENERGY_MVAH_ATTRIBUTE_TYPE),
+            MOCK_ATTRIBUTE_CONFIG_NULLABLE(ZCL_ENERGY_MVARH_ATTRIBUTE_TYPE),
```

Comment:
Does this need a "money" case?

Reply:
Just adding items by themselves will not make use of this. If we want these tested, we should add items like https://github.com/project-chip/connectedhomeip/blob/master/src/data-model-providers/codegen/tests/TestCodegenModelViaMocks.cpp#L1460

```
TestEmberScalarTypeRead<uint16_t, ZCL_ENUM16_ATTRIBUTE_TYPE>(0x1234);
// or
TestEmberScalarNullRead<OddSizedInteger<7, true>, ZCL_INT56S_ATTRIBUTE_TYPE>();
// or
TestEmberScalarTypeWrite<int32_t, ZCL_INT32S_ATTRIBUTE_TYPE>(-12345);
```

I believe originally things were copied and pasted to have all types, but we tested a subset only. We should either add more tests if we want these tested or not change these files (if we rely that storage type is tested anyway by testing the underlying type)

Reply:
OK, removed these, as well the other ones that were added back when the original energy types were added in 1.3